cns/restserver/metrics.go

```diff
@@ -10,6 +10,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/metrics"
 )
 
+const (
+	AssignIPConfigFailure   = "assignIPConfig"
+	PopulateIPConfigFailure = "populateIPConfig"
+	NoIPsAvailableFailure   = "noIPsAvailable"
```

Comment:
these label values should be `snake_case`, per Prometheus standards
```suggestion
	AssignIPConfigFailure   = "assign_ip_config"
	PopulateIPConfigFailure = "populate_ip_config"
	NoIPsAvailableFailure   = "no_ips_available"
```